include/swift/AST/DiagnosticsSema.def

```diff
@@ -2739,6 +2739,8 @@ ERROR(implements_attr_protocol_not_conformed_to,none,
 // @differentiable
 ERROR(differentiable_attr_void_result,none,
       "cannot differentiate void function %0", (DeclName))
+ERROR(differentiable_attr_inout_argument,none,
+      "cannot differentiate function %0 with inout arguments", (DeclName))
```

Comment:
```suggestion
      "cannot differentiate function %0 with respect to an 'inout' parameter", (DeclName))
```
You can differentiate a function with `inout` parameters, just not with respect to them. Also, the compiler uses the word "parameters" to denote formal parameters that a function takes.